src/database/migrations/2021_01_04_231102_create_course_rating_and_course_rating_criterias_table.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateCourseRatingAndCourseRatingCriteriasTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('course_rating_and_course_rating_criterias', function (Blueprint $table) {
+            $table->id();
+            $table->unsignedBigInteger('course_rating_criterias_id');
+            $table->unsignedBigInteger('course_ratings_id');
+            $table->enum('rating',[]);
```

Comment:
To be sure, we don't know which "ratings" are required yet and it wont be for example, a 1 - 5 criteria?

Reply:
I think you have right, I will add to array numbers from 1 to 5. Later will see.